docs/index.md

```diff
@@ -0,0 +1,165 @@
+# EMMO - Python API for the European Materials & Modelling Ontology
```

Comment:
This looks like a copy of README.md. If possible to refer to the README.md file or copy it for the documentation to have a single source of information.

Reply:
This is copied/generated from the `invoke` task `create-docs-index`, changing links to the License page and relative links from the new location. See https://github.com/emmo-repo/EMMO-python/pull/222/files#diff-2e746c9eacad6e2c2fdefdc6a665b5ce4607ea384ebf97a729044b79acc778d6R163